### DISCUSSION TITLE
Data not storing in Pavlovia

### DISCUSSION THREAD

**Discussion by FelicityF** [jspsych/jsPsych#1159] [Q&A]
My study is up on Pavlovia, but it seems my data isn't storing. When someone stops the experiment before the end of the experiment, I will be able to see the data, but when you run it till the end there is no data file. I read some things about having the pavlovia.init in the correct place (because I have an experimental timeline as well), but I'm not sure whether that might be the issue. If there is someone who could help, that would be great!

**Comment by jodeleeuw** [comment]
Hi @FelicityF,

I'm not familiar with the mechanics of running experiments on Pavlovia. I know there have been a few threads about similar issues on the forum. You may have more luck asking for help in the Pavlovia discussion group.

**Comment by sambfloyd** [comment]
Hi! I think I am having the same issue as @FelicityF - did you happen to figure this out? Thanks!
